### PR TITLE
Accelerate cache usage: combine get and contains

### PR DESCRIPTION
**Pull request by qiaojialin**

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/engine/cache/ChunkMetadataCache.java [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
 import java.io.IOException;
 import java.util.ArrayList;
+import java.util.Collections;
```
Comment (samperson1997) [inline server/src/main/java/org/apache/iotdb/db/engine/cache/ChunkMetadataCache.java:23]: useless import

File: server/src/main/java/org/apache/iotdb/db/engine/cache/ChunkCache.java [hunk 2]
```diff
@@ -93,40 +93,33 @@ public Chunk get(ChunkMetadata chunkMetaData, TsFileSequenceReader reader) throw
...
         cacheHitNum.incrementAndGet();
         printCacheLog(true);
-        Chunk chunk = lruCache.get(chunkMetaData);
         return new Chunk(chunk.getHeader(), chunk.getData().duplicate(), chunk.getDeletedAt());
       }
     } finally {
       lock.readLock().unlock();
     }
 
-    Lock cacheLock = lock.writeLock();
```
Comment (samperson1997) [inline server/src/main/java/org/apache/iotdb/db/engine/cache/ChunkCache.java:106]: useless import: `java.util.concurrent.locks.Lock`

File: server/src/main/java/org/apache/iotdb/db/engine/cache/ChunkMetadataCache.java (hunk 1 above)
Comment (qiaojialin) [reply server/src/main/java/org/apache/iotdb/db/engine/cache/ChunkMetadataCache.java:23]: fixed

File: server/src/main/java/org/apache/iotdb/db/engine/cache/ChunkCache.java (hunk 2 above)
Comment (qiaojialin) [reply server/src/main/java/org/apache/iotdb/db/engine/cache/ChunkCache.java:106]: fixed

**Pull request merged**